rootex/core/renderer/shaders/cel_pixel_shader.hlsl

```diff
@@ -0,0 +1,157 @@
+#include "register_locations_pixel_shader.h"
+#include "light.hlsli"
+#include "basic_material.hlsli"
+#include "sky.hlsli"
+
+#define NUMBEROFSHADES float(2.)
```

Comment:
better shift this to a constant buffer so that we can change this programmatically at runtime